bigml/util.py

```diff
@@ -609,11 +609,12 @@ def check_no_training_missings(input_data, fields, weight_field=None,
 
     """
     for field_id, field in fields.items():
-        if (not field_id in input_data and \
-            field['summary']['missing_count'] == 0 and \
+        if (field["optype"] != "datetime" and \
+                (not field_id in input_data and \
```

Comment:
Bad indentation... And is this parenthesis, before the `not`, needed? It's hard to understand all the conditions.

Reply:
Let me change that, but see the comment below.

Reply:
In fact, I think that's not a bad indentation. When if is multiline, there's a double indentation to differentiate the multiple lines in the condition from the first line of the if body.

Reply:
Sorry, I didn't know it! Thank you for the info!